src/services/hotstar/HotstarParser.ts

```diff
@@ -5,35 +5,36 @@ import { EpisodeItem, MovieItem } from '@models/Item';
 class _HotstarParser extends ScrobbleParser {
 	constructor() {
 		super(HotstarApi, {
-			watchingUrlRegex: /\/(?:movies|tv)\/(?<id>.+)/,
+			watchingUrlRegex: /\/(?:movies|shows)\/\D+(?<id>\d.+)\//,
 		});
 	}
 
 	parseItemFromDom() {
 		const serviceId = this.api.id;
 		const id = this.parseItemIdFromUrl();
-		const titleElement = document.querySelector('.primary-title');
+		const titleElement = document.querySelector(
+			'div#page-container>div>div>div>div>div>div>div>div>div>div.flex.flex-col>div.flex>p.ON_IMAGE.BUTTON1_MEDIUM'
```

Comment:
Can we use a more specific selector?

Reply:
I think chaining selector is best way to get the element. Because in the new website no more id/class in specific. When i've only use `p.ON_IMAGE.BUTTON1_MEDIUM` selector, will get content rating that show and disapear in a few seconds. And when use class like `_2Xo-ONKnvHxYr5Qupt1eXf`, i think will change regulary when site has new update because the class is generated by library they use. 

![image](https://github.com/trakt-tools/universal-trakt-scrobbler/assets/18456011/c8cdf756-bcaa-4b91-9fd5-900412b37586)